saleor/graphql/page/bulk_mutations.py

```diff
@@ -19,6 +23,22 @@ class Meta:
         error_type_class = PageError
         error_type_field = "page_errors"
 
+    @classmethod
+    def perform_mutation(cls, _root, info, ids, **data):
+        try:
+            pks = cls.get_global_ids_or_error(ids, only_type=Page, field="pk")
+        except ValidationError as error:
+            return 0, error
```

Comment:
we could add test for scenario with validation error

Reply:
What is `0` ? It's a pk or `error` value? Wouldn't be better just raise Validation error?
@maarcingebala @korycins

Reply:
For bulk mutations, we return also information about the number of returned items, even if the mutation failed, because of that we are returning 0 and error here.